src/dropdown/dropdown.spec.ts

```diff
@@ -190,6 +190,30 @@ describe('ngb-dropdown-toggle', () => {
     expect(buttonEl.getAttribute('aria-expanded')).toBe('false');
   });
 
+  it('should toggle dropdown on click of child of toggle', () => {
```

Comment:
phrasing confused me a little when I first read it, but can't come up with a better one :)